app/models/api_model.php

```diff
@@ -66,8 +66,9 @@
             // Include the current template
             require TEMPLATES . $project->getTemplate($store, $channel);
 
-            // Google has a 500 characters limit for the What's New section
-            if ($store == 'google' && $set_limit(500, $whatsnew($translations))) {
+            // Only Google has a 500 characters limit for the What's New section
+            if ($store == 'google' && $set_limit(500, $whatsnew($translations))
+                || $store == 'apple') {
```

Comment:
I would prefer a switch. Note that you don't need to split the condition, the 80 character limit is not a PHP rule, in PHP 80 is the advised length but the upper limit is 120. In this case I think it's more readable to have an if statement on a 100 characters line than one on 2 lines.